incubator/@react-native-webapis/battery-status/RNWBatteryStatus.podspec

```
@@ -5,7 +5,7 @@ version = package['version']
 repository = package['repository']
 
 Pod::Spec.new do |s|
-  s.name      = 'RNWBatteryStatus'
+  File.basename(__FILE__, '.podspec')
```

I completely missed this 🤦‍♂️:
```suggestion
  s.name      = File.basename(__FILE__, '.podspec')
```